layouts/partials/widgets/social.html

```diff
@@ -49,6 +49,14 @@ <h4 class="widget-social__title widget__title">{{ T "social_title" }}</h4>
 			</a>
 		</div>
 		{{- end }}
+		{{- with .Site.Params.widgets.social.bitbucket }}
```

Comment:
1. Delete extra path in middle of icon (`M101.272`...)
1. Fix common fill attribute in svg tag (`fill="fff"` -> `fill="#fff"`)
1. Delete `fill`attributes in paths

Reply:
I've made the changes. Thanks for your guidance.

Reply:
Extra path (`M244.61`...) still exist